UrbanMobility/Stop/doc/spec.md

```diff
@@ -0,0 +1,108 @@
+# gtfs:Stop
+
+## Description
+
+See [https://developers.google.com/transit/gtfs/reference/#stopstxt](https://developers.google.com/transit/gtfs/reference/#stopstxt)
+
+It represents a GTFS `stop` which `location_type` shall be equal to `0`.
+
+## Data Model
+
++ `id`: Entity Id
+  + It shall be `urn:ngsi-ld:gtfs:Stop:<stop_identifier>` being `stop_identifier` a value that can derived from the GTFS `stop_id` field. 
+
++ `type`: Entity Type 
+  + It shall be equal to `gtfs:Stop`
+ 
++ `dateCreated` : Entity's creation timestamp.
+  + Attribute type: [DateTime](https://schema.org/DateTime)
+  + Read-Only. Automatically generated. 
+  
++ `dateModified` : Last update timestamp of this Entity.
+  + Attribute type: [DateTime](https://schema.org/DateTime)
+  + Read-Only. Automatically generated.
+  
++ `name`: Same as GTFS `stop_name`. 
+  + Attribute type: Property. [Text](https://schema.org/Text)
+  + Mandatory
+  
++ `code`: Same as GTFS `stop_code`. 
+  + Attribute type: Property. [Text](https://schema.org/Text)
+  + Optional
+  
++ `page`: Same as GTFS `stop_url`. 
+  + Attribute type: Property. [URL](https://schema.org/URL)
+  + Optional
+  
++ `description`: Same as GTFS `stop_desc`. 
+  + Attribute type: Property. [Text](https://schema.org/Text)
+  + Optional
+ 
++ `location`: Stop's location encoded as GeoJSON Point which coordinates shall be in the form [`stop_long`,`stop_lat`].
+  + Attribute type: GeoProperty. `geo:json`.
+  + Normative References: [rfc7946](https://tools.ietf.org/html/rfc7946)
+  + Mandatory
+
++ `wheelChairAccessible`: Same as GTFS `wheelchair_boarding`. 
+  + Attribute type: Property. [Text](https://schema.org/Text)
+  + Allowed values: (`0`, `1`, `2`) as per the [GTFS](https://developers.google.com/transit/gtfs/reference/#stoptxt)
```

Comment:
if it is always a number, why text?

Reply:
the GTFS spec does not say anything specific about being it a number, so given that it can only take these values, I would prefer it to be just a string, to be safer, future-wise